app/src/main/java/com/framgia/forder/screen/shopinfo/ShopinfoViewModel.java

```diff
@@ -1,14 +1,25 @@
 package com.framgia.forder.screen.shopinfo;
 
+import com.framgia.forder.data.model.CollectionAvatar;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.CollectionAvatar. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports